Dockerfile.ocp

```diff
@@ -1,9 +1,10 @@
 FROM ubi8
 
 RUN yum update -y && \
-    yum install -y openstack-ironic-api openstack-ironic-conductor crudini \
+    yum install -y python3-gunicorn openstack-ironic-api openstack-ironic-conductor crudini \
         iproute iptables dnsmasq httpd qemu-img parted gdisk ipxe-bootimgs psmisc procps-ng \
-        mariadb-server python2-chardet ipxe-roms-qemu genisoimage && \
+        mariadb-server ipxe-roms-qemu genisoimage && \
+    yum install -y python3-ironic-prometheus-exporter && \
```

Comment:
any reason for a separate yum install for this one?

Reply:
misprint from metal3, I'll merge the  command with the one above